WebPhapp/backend/server.js

```diff
@@ -123,6 +134,50 @@ app.get('/api/v1/patients', (req,res) => {
     res.json(matchingPatients);
 });
 
+/*
+About:
+    An api endpoint that returns the info about a patient given a specific
+    patient ID. Patient data temporarily includes date of birth, first and 
+    last name, and patient ID. 
+    TODO: restrict query to a single prescriber.
+Examples:
+    Directly in terminal:
+        >>> curl "http://localhost:5000/api/v1/patients/01"
```

Comment:
I really appreciate that you put the `curl` command in here! So, thanks for that.

Reply:
But, personally do not like the quotes around the curl statement. I do not see that with other curl code that I read. Could you change this?

Reply:
@mdulin2 I have been putting quotes around the curl statement because in many requests, query parameters and tokens are used that must be escaped or else will be interpreted as bash commands in the terminal. By placing quotes, you do not need to escape every single character that needs escaping (think: \&). If you still think that quotes should be removed, let me know and I will do so!